firecracker-vms.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please remove this file

Reply:
Deleted in 112c9894452b36afc9716ffd7aa2514800d357a8.